services/sensorCachingService.js

```diff
@@ -0,0 +1,84 @@
+//Declaring sensors and creating the function(s) for it
+var sensors = {}
+
+/**
+ *Retrieve the last known temprature/humidity form a specific sensor
+ * @author Lars
+ * @param {entry} entry The sensor entry that you want to add to the cache
+ */
+function retrieveLastValue(entry) {
+    const key = getKey(entry);
+    if(sensors[key]){
+        return sensors[key].value
+    }
+    return null
+}
+
+/**
+ * Store the current value in the sensors obj
+ * @author Lars
+ * @param {entry} entry The sensor entry that you want to add to the cache
+ */
+function storeCurrentValue(entry){
+    sensors[getKey(entry)] = entry
+}
+
+/**
+ * Retrieve the maximum temprature/humidity for a specific floor.
+ * @author Lars
+ * @param {floor} floor The floor you want to recieve the max temp of.
+ */
+function getMaxTemp(floor){
+    var highestTemp
+    for(var element of sensorsFromFloor(floor)){
+        if(!highestTemp || element.value > highestTemp){
+            highestTemp = element.value
+        }
+    }
+    return highestTemp
+}
+
+/**
+ * Retrieve the minimum temprature/humidity for a specific floor.
+ * @author Lars
+ * @param {floor} floor The floor you want to recieve the min temp of.
+ */
+function getMinTemp(floor){
+    var lowestTemp
+    for(var element of sensorsFromFloor(floor)){
+        if(!lowestTemp || element.value < lowestTemp){
+            lowestTemp = element.value
+        }
+    }
+    return lowestTemp
+}
+
+module.exports = {retrieveLastValue, storeCurrentValue, getMaxTemp, getMinTemp}
+
+
+/**
+ * Check required fields and return the key used to retrieve/store in the sensors obj
+ * @author Lars
+ * @param {entry} entry The whole sensor entry
+ */
+function getKey(entry){
```

Comment:
Since we're here, this name could be a bit more descriptive, something along the lines of `generateCacheKey` or `getEntryIdentifier` or something similar